### DISCUSSION TITLE
Feature request: add an rpc method for obtaining MEV rewards

### DISCUSSION THREAD

**Discussion by thegostep** [flashbots/pm#46] [miners]
Currently, the only way for miners to determine the amount of value received from FlashBots is to look on chain, look at their node logs or query https://blocks.flashbots.net

It has been requested to create an rpc endpoint which would return the MEV value obtained by the coinbase address for a given block: https://github.com/flashbots/mev-geth/issues/24

**Comment by Poluect** [comment]
A similar solution but probably the one that is easier to implement can be extending either `eth_getWork`, or `eth_submitBlock` methods (or in case of `eth_submitBlock` it may be better to implement own submit method), so that a response contains a coinbase reward for the given work / block

**Comment by tkstanczak** [comment]
Problems to solve:
* either store in the database or detect the MEV bundles included in the block

Then can do sth like debug_traceBlock but make two additional calls to ask about the coinbase balance before and after the bundle.

**Reply by tkstanczak** [reply]
might be easier to start with total block coinbase payment so MEV bundles do not have to be detected
then can subtract gas payments and reward
